src/Components/Map/Map.js

```diff
@@ -0,0 +1,23 @@
+import { useRef } from "react";
+import "./Map.css";
+const AutoComplete = () => {
+//  const autoCompleteRef = useRef();
+ const inputRef = useRef();
+//  const options = {
+//     fields: ["formatted_address", "geometry", "name"],
+//     strictBounds: false,
+//     types: ["establishment"]
+//   };
+//  useEffect(() => {
+//   autoCompleteRef.current = new window.google.maps.places.Autocomplete(
+//    inputRef.current,
+//    options
+//   );
+//  }, []);
+ return (
+  <div>
+   <input className="long-input" ref={inputRef} />
+  </div>
+ );
+};
+export default AutoComplete;
```

Comment:
Appreciate you pausing this! I wanna make sure we implement this seamlessly so would rather wait and come back when we have bandwidth.